openapi/neutrino/client.bal

```diff
@@ -18,333 +18,449 @@ import ballerina/http;
 
 # Provides API key configurations needed when communicating with a remote HTTP endpoint.
 public type ApiKeysConfig record {|
-    # API keys related to connector authentication
-    map<string> apiKeys;
+    # Represents API Key `user-id`
+    string userId;
+    # Represent API Key `api-key`
+    string apiKey;
 |};
 
 # This is a generated connector for [Neutrino API v3.5.0](https://www.neutrinoapi.com/api/api-basics/) OpenAPI specification.
 # The Neutrino API is a general-purpose API.
 @display {label: "Neutrino", iconPath: "resources/neutrino.svg"}
 public isolated client class Client {
     final http:Client clientEp;
-    final readonly & map<string> apiKeys;
+    final readonly & ApiKeysConfig apiKeyConfig;
     # Gets invoked to initialize the `connector`.
     # The connector initialization requires setting the API credentials. 
     # Create a [Neutrino API account](https://www.neutrinoapi.com/signup) and obtain tokens by following [this guide](https://www.neutrinoapi.com/api/api-basics/).
     #
-    # + apiKeyConfig - Provide your API keys as `user-id` and `api-key`. Eg: `{"user-id" : "<User ID>", "api-key" : "<API key>"}` 
+    # + apiKeyConfig - API keys for authorization 
     # + clientConfig - The configurations to be used when initializing the `connector` 
     # + serviceUrl - URL of the target service 
     # + return - An error if connector initialization failed 
     public isolated function init(ApiKeysConfig apiKeyConfig, http:ClientConfiguration clientConfig =  {}, string serviceUrl = "https://neutrinoapi.net/") returns error? {
         http:Client httpEp = check new (serviceUrl, clientConfig);
         self.clientEp = httpEp;
-        self.apiKeys = apiKeyConfig.apiKeys.cloneReadOnly();
+        self.apiKeyConfig = apiKeyConfig.cloneReadOnly();
+    }
+    # Image Resize
+    #
+    # + payload - Request payload 
+    # + return - Response status 
+    remote isolated function imageResize(ImageresizeBody payload) returns string|error {
```

Comment:
Wondering whether there was a reason not to generate these functions at the first time ?

Reply:
@kapilraajP Could you remember the reason to remove these operations when sending the initial PR ?

Reply:
@RolandHewage

Reply:
These operations are related to the `Imaging` tag. Since these operations doesn't seems to be important we didn't generate these operations initially. I would suggest not to generate those operations since they are image related ones.

Reply:
Commented the operation in the openapi.yaml file